care/facility/api/serializers/asset.py

```diff
@@ -30,7 +30,7 @@ class AssetSerializer(ModelSerializer):
     id = UUIDField(source="external_id", read_only=True)
     status = ChoiceField(choices=Asset.StatusChoices, read_only=True)
     asset_type = ChoiceField(choices=Asset.AssetTypeChoices)
-
+    is_working = ChoiceField(choices=Asset.AssetWorkingChoices)
```

Comment:
Does it make more sense for the BE to return True/False rather than text? It is usually a pattern that is followed to decouple texts from the BE to allow easier translations.

Reply:
sure, I'll change it to text.

Reply:
move to boolean values.